torch/testing/_internal/common_nn.py

```diff
@@ -4695,6 +4720,36 @@ def padding3d_circular(input, pad):
         desc='dim_is_3',
         check_bfloat16=False,
     ),
+    dict(
+        module_name='CrossEntropyLoss',
+        input_size=(5, 3),
+        target_fn=lambda: torch.rand(5, 3).softmax(dim=1),
+        reference_fn=lambda i, t, m:
+            loss_reference_fns['CrossEntropyLoss'](i, t, reduction=get_reduction(m)),
+        check_sum_reduction=True,
+        desc='2d_prob_target',
+        check_bfloat16=False,
+    ),
```

Comment:
We should test that `weights` work as expected with a probability target; I don't see weights being used by these tests

Reply:
The one-hot test in `test_nn.py` uses weights but I'll also add them here.